vulkaninfo/vulkaninfo.h

```diff
@@ -715,9 +714,90 @@ struct AppInstance {
         global_extensions = AppGetGlobalLayerExtensions(nullptr);
     }
     void AppCompileInstanceExtensionsToEnable() {
-        // Get all supported Instance extensions (excl. layer-provided ones)
         for (const auto &ext : global_extensions) {
-            inst_extensions.push_back(ext.extensionName);
+            if (strcmp(VK_EXT_DEBUG_REPORT_EXTENSION_NAME, ext.extensionName) == 0) {
```

Comment:
I have a slight preference for putting the extension names in an array, up to you though.

Reply:
Array, like a std::array? Not sure how to handle dynamicism using a std::array.

Reply:
Yeah I suppose it would have to be something like a vector that can handle #ifdefs in the initializer list. Or you could a braced init list directly in a range based for loop. Maybe more trouble than it is worth.